soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -0,0 +1,2044 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef CHIP_CHIPREGS_H
+#define CHIP_CHIPREGS_H
+
+#define EC_Register_Base_Address    0x00F00000
+#define BIT_SHIFT(bit, sft) (bit << sft)
+
+#define BIT0 	0x00000001
+#define BIT1 	0x00000002
+#define BIT2 	0x00000004
+#define BIT3   	0x00000008
```

Comment:
Delete extra spaces. Either have a consistent number of spaces between the BITx and the 0xvalue (and that should be only 1 space), or line up all the 0xvalues on the same column.